app-creator/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/labstack/echo"
+)
+
+func main() {
+	// Creating echo object.
+	e := echo.New()
+
+	// Setting static directory.
+	e.Static("/static", "static")
+
+	// Serving index.html on "/" route.
+	e.File("/", "static/index.html")
+
+	// Check if a PORT env variable is set.
+	port := os.Getenv("PORT")
+	if port == "" {
```

Comment:
Add a regex check for the port value to be all numbers and always remove leading colons if they exist. https://golang.org/pkg/regexp/

Reply:
Also it would be good to add tests for this too.